pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/Annotateable.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.ast;
+
+import java.util.Collection;
+import java.util.List;
+
+public interface Annotateable extends JavaNode {
```

Comment:
Pretty sure this should be called `Annotatable` ([with no 'e'](https://english.stackexchange.com/questions/76043/rules-for-removing-last-vowel-when-adding-able#76044))